heron/stmgr/src/cpp/manager/stmgr-server.cpp

```diff
@@ -107,6 +110,11 @@ StMgrServer::StMgrServer(EventLoop* eventLoop, const NetworkOptions& _options,
   metrics_manager_client_->register_metric(METRIC_TIME_SPENT_BACK_PRESSURE_INIT,
                                            back_pressure_metric_initiated_);
   spouts_under_back_pressure_ = false;
+
+  // Update queue related metrics every 10 seconds
+  CHECK_GT(eventLoop_->registerTimer([this](EventLoop::Status status) {
+    this->UpdateStMgrServerMetrics(status);
+  }, true, QUEUE_METRICS_FREQUENCY), 0);
```

Comment:
It is just a copy of PROCESS_METRICS_FREQUENCY. I understand you may suggest to expose it in config file, but we already have too many config's...